### PR TITLE
feat(docker): Add `lazydocker.nvim`

### PR DESCRIPTION
**Pull request by Mourishitz**
## 📑 Description

Adds [Lazy Docker](https://github.com/mgierada/lazydocker.nvim) package to the AstroNvim community.

## ℹ Additional Information

This PR adds a new shortcut to `Leader + t` + `d` which runs the Lazy Docker plugin.
It depends only on [lazydocker.nvim](https://github.com/mgierada/lazydocker.nvim) and it uses built-in ToggleTerm support.

No breaking changes or old behavior interference.

image

### CODE REVIEW

File: lua/astrocommunity/docker/lazydocker/init.lua [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+              [prefix .. "d"] = {
+                function() require("lazydocker").open() end,
+                desc = "ToggleTerm LazyDocker",
+              },
+            },
+          },
+        },
+      },
+    },
+    config = function() require("lazydocker").setup {} end,
```
Comment (Uzaaft) [inline lua/astrocommunity/docker/lazydocker/init.lua:21]:
```suggestion
    config = function() require("lazydocker").setup {} end,
```
This is not needed. Use opts instead.

**Review by Uzaaft — CHANGES_REQUESTED** [review]
See comment

**Comment by Mourishitz** [comment]
Got it!! Tested it out and still runs smooth and perfect! Thanks for the suggestion

File: lua/astrocommunity/docker/lazydocker/init.lua [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                desc = "ToggleTerm LazyDocker",
+              },
+            },
+          },
+        },
+      },
+    },
+    opts = {
+      function() require("lazydocker").setup() end,
+    },
```
Comment (Uzaaft) [inline lua/astrocommunity/docker/lazydocker/init.lua:23]:
```suggestion
    opts = {},
```

File: lua/astrocommunity/docker/lazydocker/init.lua [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  {
+    "mgierada/lazydocker.nvim",
+    dependencies = {
+      "akinsho/toggleterm.nvim",
+      {
+        "AstroNvim/astrocore",
+        opts = {
+          mappings = {
+            n = {
+              [prefix .. "d"] = {
```
Comment (Uzaaft) [inline lua/astrocommunity/docker/lazydocker/init.lua:12]:
```suggestion
              ["<Leader>t" .. "d"] = {
```

File: lua/astrocommunity/docker/lazydocker/init.lua [hunk 4]
```diff
@@ -0,0 +1,23 @@
+local prefix = "<Leader>t"
```
Comment (Uzaaft) [inline lua/astrocommunity/docker/lazydocker/init.lua:1]:
```suggestion
```

**Comment by Uzaaft** [comment]
You still didnt use it correctly. Test out the new plugin spec @Mourishitz

**Comment by Mourishitz** [comment]
Yes it seems to work really well! Thanks for the suggestions, its my first contribution to astro community and I'm not used to the best practices yet

**Review by Uzaaft — APPROVED** [review]

**Pull request merged**